docs/commands/json.mset.md

```diff
@@ -1 +1,60 @@
-Sets or updates the JSON value of one or more keys.
+Set or updates JSON values of one or more keys (multiple) for a given tripplet `key-path-value`
+
+JSON.MSET is atomic, so all given keys are set at once or not. It is not possible for clients to see that some of the keys were updated while others are unchanged.
```

Comment:
JSON.MSET is atomic, hence, all given additions or updates are either applied or not.

Reply:
OK